src/lib/index.ts

```diff
@@ -0,0 +1,38 @@
+export {
+  BaseAsyncIterReader,
+  AsyncIterReader,
+  LimitReader,
+  NoConcatInflator,
+} from "./readers";
+export type { AsyncIterReaderOpts } from "./readers";
+
+export { StatusAndHeadersParser, StatusAndHeaders } from "./statusandheaders";
+
+export { WARCParser } from "./warcparser";
+export type { WARCParserOpts } from "./warcparser";
+
+export { WARCSerializer } from "./warcserializer";
+export type { WARCSerializerOpts } from "./warcserializer";
+
+export { WARCRecord, WARC_1_1, WARC_1_0 } from "./warcrecord";
+export type { WARCRecordOpts, WARCType } from "./warcrecord";
+
+export { Indexer, CDXIndexer } from "./indexer";
+
+export {
+  postToGetUrl,
+  getSurt,
+  appendRequestQuery,
+  jsonToQueryString,
```

Comment:
Need to include all of the utils exports (at least for now, as they're used in other libs that use warcio.js)
```suggestion
  jsonToQueryParams,
  mfdToQueryParams,
  jsonToQueryString,
  splitChunks,
```